browser/src/neovim/NeovimInstance.ts

```diff
@@ -760,6 +765,7 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
                     break
                 case "scroll":
                     this.emit("action", Actions.scroll(a[0][0]))
+                    this._dispatchScrollEvent()
```

Comment:
@bryphe ah this was the line I was referring to which I though represented a scroll event from neovim but that was the what I was logging out which was providing a more accurate response to scrolling didnt realise it was from the redraw event

Reply:
Haha ya, I think the "bug" is that we weren't using it for our `onScroll` event 😄 So we were missing out on this info, which is the most reliable way to check for scroll events when the cursor doesn't move (it misses large scroll events, like `G` on a long file)